language-server/compile-language-server.sh

```diff
@@ -10,21 +10,29 @@ SRC_DIR="${REPO_DIR}/${GITHUB_REPO_NAME}"
 DIST_DIR="${REPO_DIR}/out"
 
 
+# -------- #
+# clean up #
+# -------- #
+
+rm -rf \
+    "${DIST_DIR}" \
+    "package.json" "package-lock.json"
```

Comment:
This script should work the same regardless of the working directory (if called from the root of the package or `language-server` dir. I think this tries to remove package*.json from the working directory.

Reply:
Ah, yeah, that's not intentional :P